src/ncp/ncp_base.cpp

```diff
@@ -55,6 +55,43 @@ namespace Ncp {
 // MARK: Utility Functions
 // ----------------------------------------------------------------------------
 
+uint8_t NcpBase::InstanceToIid(Instance *aInstance)
+{
+    uint8_t index = 0;
+
+    OT_UNUSED_VARIABLE(aInstance);
+
+#if OPENTHREAD_CONFIG_MULTIPAN_RCP_ENABLE && OPENTHREAD_RADIO
+    index = SPINEL_HEADER_GET_IID(SPINEL_HEADER_IID_BROADCAST); // use broadcast if no match
+
+    for (int i = 0; i < kSpinelInterfaceCount; i++)
+    {
+        if (aInstance == mInstances[i])
+        {
+            index = i;
+            break;
+        }
+    }
+#endif // OPENTHREAD_CONFIG_MULTIPAN_RCP_ENABLE  && OPENTHREAD_RADIO
+
+    return index;
+}
+
+Instance *NcpBase::IidToInstance(uint8_t aIid)
+{
+    Instance *instance;
+    OT_ASSERT(aIid < kSpinelInterfaceCount);
+
+#if OPENTHREAD_CONFIG_MULTIPAN_RCP_ENABLE && OPENTHREAD_RADIO
+    instance = mInstances[aIid];
+#else
+    OT_UNUSED_VARIABLE(aIid);
+    instance = mInstance;
+#endif // OPENTHREAD_CONFIG_MULTIPAN_RCP_ENABLE  && OPENTHREAD_RADIO
+
+    return instance;
+}
```

Comment:
It would be useful to have the following utility functions. Can we add in this PR?  OR would you prefer in a separate PR?

extern "C" otInstance* otIidToInstance(uint8_t aIid)
{
    return ot::Ncp::NcpBase::GetNcpInstance()->IidToInstance(aIid);
}

extern "C" uint8_t otInstanceToIid(otInstance *aInstance)
{
    return ot::Ncp::NcpBase::GetNcpInstance()->InstanceToIid(static_cast<ot::Instance *>(aInstance));
}

Reply:
added

Reply:
Thank you.